### PR TITLE
Fix logging coming from luet

### LINKED ISSUE #303: Log line missing space during "build-iso"
Running the `elemental-iso-script` from the `elemental` repo produces log lines in need of an additional space, like

```
Downloadingquay.io/costoolkit/releases-green:repository.yaml
```

### PR DESCRIPTION
**Pull request by Itxaka**
transform and clean the args received from luet logging in order to
print decent messages

Fixes #303 

Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

**Comment by Itxaka** [comment]
```
INFO[2022-08-09T18:16:15+02:00] Starting elemental version v0.0.15           
INFO[2022-08-09T18:16:15+02:00] Preparing squashfs root...                   
INFO[2022-08-09T18:16:15+02:00] Copying isoroot source...                    
INFO[2022-08-09T18:16:15+02:00] Finished copying isoroot into /tmp/elemental-iso614776587/rootfs 
INFO[2022-08-09T18:16:15+02:00] Preparing EFI image...                       
INFO[2022-08-09T18:16:15+02:00] Copying live/grub2-efi-image source...       
INFO[2022-08-09T18:16:15+02:00] Downloading quay.io/costoolkit/releases-green:repository.yaml 
INFO[2022-08-09T18:16:16+02:00] Pulled: sha256:e4581108408c1a5412e1e9982e326dd9c02d58eb45dd6e7a71808a1188bf2dec 
INFO[2022-08-09T18:16:16+02:00] Size: 628B                                   
INFO[2022-08-09T18:16:16+02:00] Downloading quay.io/costoolkit/releases-green:tree.tar.zst 
INFO[2022-08-09T18:16:17+02:00] Pulled: sha256:35d03fbd3c6aeb55f752e6a410ad79438976d7bf4fb13db9bf3da19ea2464a56 
INFO[2022-08-09T18:16:17+02:00] Size: 8.12KiB                                
INFO[2022-08-09T18:16:17+02:00] Downloading quay.io/costoolkit/releases-green:repository.meta.yaml.tar.zst 
INFO[2022-08-09T18:16:17+02:00] Pulled: sha256:96b9ae821ec6eedab6a7e4a63c4abbcccf653a04009e7ab7d737dfe92b96b474 
INFO[2022-08-09T18:16:17+02:00] Size: 163.9KiB                               
INFO[2022-08-09T18:16:17+02:00] Repository cOS revision: 420 (2022-08-09 15:44:49 +0200 CEST) 
INFO[2022-08-09T18:16:17+02:00] Repository: b0e1eae606fa226d19149b8f9f372ab7 Priority: 90 Type: docker 
INFO[2022-08-09T18:16:17+02:00] Packages that are going to be installed in the system: 

Program Name                 | Version | License | Repository                      
live/grub2-efi-image-0.0.2-2 | 0.0.2-2 |         | b0e1eae606fa226d19149b8f9f372ab7

Downloading packages [0/1] █                                                                       0% | 0s
INFO[2022-08-09T18:16:17+02:00] Downloading image quay.io/costoolkit/releases-green:grub2-efi-image-live-0.0.2-2 
INFO[2022-08-09T18:16:18+02:00] Image: quay.io/costoolkit/releases-green:grub2-efi-image-live-0.0.2-2. Pulled: sha256:18464770ce888708c55617ca518acf46c2f49d4b30f63f7ff5b8c75818b0f84a. Size: 1.162MiB 

```

**Comment by Itxaka** [comment]
lint issues fixed on https://github.com/rancher/elemental-cli/pull/308

File: pkg/types/v1/logger.go [hunk 1]
```diff
@@ -116,14 +117,54 @@ func (w *logrusWrapper) Copy() (types.Logger, error) {
...
+	w.Logger.Error(converted)
+}
+
+func (w *logrusWrapper) Fatal(args ...interface{}) {
+	converted := convert(args)
+	w.Logger.Fatal(converted)
+}
+
+// convert changes a list of interfaces into a proper joined string ready to log
+func convert(args []interface{}) string {
```
Comment (fgiudici) [inline pkg/types/v1/logger.go:153]: having a function to convert each log from luet doesn't look great... 🤔
May be possible to stick to luet logging and move to a different log (dropping logrus) to keep also emoji if present? 🤔

**Review by fgiudici — APPROVED** [review]
Not 100% happy to have a function to convert logs... but quite happy that this gets rid of the messy logging we actually have (missing spaces)! Waaaay better!
Maybe we can consider to rework the logging to stick with luet format 🤔 ... something for another PR btw.

**Comment by Itxaka** [comment]
Would you believe me if I tell you that this is similar to what luet does? They need some manipulation of the logs to be able to log them cleanly supporting emojis and screen types and so on.

So even if we adapted our log stuff to their format, there would be a more complex conversion done down the line 😭

**Pull request merged**